src/deloader/deloader.cpp

```diff
@@ -45,20 +45,11 @@ static bool verify_plugin(const k4a_plugin_t *plugin)
 {
     RETURN_VALUE_IF_ARG(false, plugin == NULL);
 
-    LOG_INFO("Loaded K4A Plugin with version: %u.%u.%u",
+    LOG_INFO("Loaded Depth Engine version: %u.%u.%u",
              plugin->version.major,
              plugin->version.minor,
              plugin->version.patch);
 
-    // Major versions must match
-    if (plugin->version.major != K4A_PLUGIN_MAJOR_VERSION)
-    {
-        LOG_ERROR("Plugin Major Version Mismatch. Expected %u. Found %u",
-                  K4A_PLUGIN_MAJOR_VERSION,
-                  plugin->version.major);
-        return false;
-    }
-
```

Comment:
Our goal is that the number in the file name is our "guarantee" that the file is compatible?

Reply:
yes